CONTRIBUTING.md

```diff
@@ -82,49 +82,49 @@ There are multiple ways to add or edit your translation of the game:
 
 ## Using a GitHub fork
 
-[Fork the repository](https://github.com/ReactiveDrop/reactivedrop_translations/fork), edit & commit it and then push your changes back to the origin.
+[Fork the repository](https://github.com/ReactiveDrop/reactivedrop_translations/fork), edit & commit it, then push your changes back to the origin.
 
 Use an editor to make your changes and integrate them through github. Read more about it [on our wiki](https://github.com/ReactiveDrop/reactivedrop_translations/wiki).
 
 ### Pre-requisites
-- Any unicode plain text editor. E.g. [Notepad++](https://notepad-plus-plus.org/) or [Visual Studio Code](https://code.visualstudio.com/).
+- Any unicode plain text editor. Eg. [Notepad++](https://notepad-plus-plus.org/) or [Visual Studio Code](https://code.visualstudio.com/).
 - An [GIT installation](https://git-scm.com/downloads)
 
 # Files
 
 ## File Encoding
-Please make sure that your text editor preserves file's encoding.  
-The new files you create need to have same encoding as their english counterpart (e.g. labsmail1_czech.txt needs to be UTF-8 encoded just like labsmail1_english.txt). If you use Notepad++ you can see the file's encoding in the menu Encoding.
+Make sure your text editor preserves file encodings.  
+New files need to have the same encoding as their english counterparts (eg. labsmail1_czech.txt needs to be UTF-8 encoded just like labsmail1_english.txt). If you use Notepad++ you can see the file's encoding in the menu Encoding. vscode shows the files encoding in its status bar.
 
 ## File type
 ### Achievements
 These files are automatically created based on the `resource/reactivedrop_*.txt` files. You can manually trigger an update of the files through executing `utils/translation-sync-tool.exe`.
 * sync-tool: Very useful verification tool!
- 1. It can help solving issues with ci-checks. After you have manually modified the content about achievements, it is highly recommended to run the tool to check for problems. Avoid Pull Request Failure.
- 2. It can automatically synchronize achievement fields from `reactivedrop_*.txt` to `563560_loc_*.vdf`! This is very useful to avoid the painful task of comparing and duplicating work.
+ 1. It can help solving issues with ci-checks. After having manually modified the content of achievements, it is highly recommended to let the tool check for problems.
+ 2. It can automatically synchronize achievement fields from `reactivedrop_*.txt` to `563560_loc_*.vdf`. This can be very useful to avoid the time-consuming task of comparing duplicated work.
```

Comment:
Achievements are now synced post pull request like the PROGRESS.md file is.

Reply:
I don't get how I could write it differently. It's still an ci-check, or is ist? Because the only difference I can see is that it automatically executes after the translators commit-creation instead of pr-creation. And prs and commits aren't even mentioned in this part.

Reply:
I tried to make it more clear, here https://github.com/ReactiveDrop/reactivedrop_translations/pull/1295